src/theme/default.css

```diff
@@ -52,6 +52,13 @@ html[data-theme='default'] {
 
   /* Links (a tag) */
   --link: rgba(143, 154, 255, 1);
+
+  --v-charcoal: rgba(37, 33, 30, 1);
+  --warm-gray: rgba(172, 163, 157, 1);
+  --dark-gray: rgba(55, 50, 47, 1);
+  --l-black: rgba(23, 20, 18, 1);
+
+  --header-height: 5.25rem;
```

Comment:
Does this has to exist in the theme or in global?

Reply:
this variable is used across several components, that's why I put it here. And it remains the same in different themes, so I don't consider it should go to any theme